robotis-op2/robot-window.md

```diff
@@ -0,0 +1,18 @@
+# Robot Window
+
+When you double click on the robot, a new window appears. This window is called
+`Robot Window`, it has several tabs allowing you to perform different things.
```

Comment:
Note that in the User Guide where the robot window is described no capital letters are used